--- plugins/org.eclipse.xtext/src-gen/org/eclipse/xtext/parser/antlr/internal/InternalXtextLexer.java
+++ plugins/org.eclipse.xtext/src-gen/org/eclipse/xtext/parser/antlr/internal/InternalXtextLexer.java
@@ -10,6 +10,9 @@ import java.util.Stack;
 import java.util.List;
 import java.util.ArrayList;
 
+/**
+ * @since 2.9
+ */
 public class InternalXtextLexer extends Lexer {
     public static final int RULE_ID=5;
     public static final int T__29=29; 
