
--- plugins/org.eclipse.xtext.xtext.ui/src-gen/org/eclipse/xtext/ui/contentassist/antlr/XtextParser.java
+++ plugins/org.eclipse.xtext.xtext.ui/src-gen/org/eclipse/xtext/ui/contentassist/antlr/XtextParser.java
@@ -16,7 +16,9 @@ import org.eclipse.xtext.ui.editor.contentassist.antlr.internal.AbstractInternal
 import com.google.inject.Inject;
 
 import org.eclipse.xtext.services.XtextGrammarAccess;
+/**
+ * @since 2.9
+ */
-
 public class XtextParser extends AbstractContentAssistParser {
 	
 	@Inject 
